05/app.js

```diff
@@ -8,7 +8,25 @@ const stats = {
 };
 
 /* tutaj umieść swój kod */
-
+const pList = document.querySelectorAll('p');
+pList.forEach(function(item) {
+    item.addEventListener('click', function(e) {
+        e.preventDefault();
+        if(e.target.tagName === "A" && e.target.href.includes('/dolor.html')) {
+            stats.links[e.target.getAttribute('href')] += 1;
+            console.log(stats);
+        } else if(e.target.tagName === "P") {
```

Comment:
Ogólnie najpierw bym sprawdził czy jest to element p (jesli nie to pewnie jest to a).
Następnie wykonał odpowiednie operacje w zależności od typu elementu.